docs/kafka/topic-configuration-kafka/README.adoc

```diff
@@ -282,7 +280,7 @@ h|Kafka property name
 
 === Messages
 
-These properties control how your messages are handled in the Kafka instance.
+The following topic properties control how your messages are handled in the Kafka instance.
```

Comment:
I wonder if we can eliminate passive voice from this and a couple of the subsequent intro sentences? Per 2.4 [here](https://docs.google.com/document/d/1TERH4XRH6La44hUQH88LUaax-35JH3dLXmceGJoLv7I/edit#) we cautioned against not using active voice if the agent of an action is unknown or unimportant, when it makes a sentence awkward or clunky, or when it introduces anthropomorphism. But I'm not sure that would be the case with a couple of these.

For example, it seems reasonable to say "The following topic properties control how the Kafka instance handles messages" and "The following topic properties define how the Kafka instance handles your log."

WDYT?

Reply:
Good point. I reviewed all intro sentences and revised the ones with passive voice to be active. I also reviewed briefly the property descriptions as well, and many of them use passive too, but imo, revising those ones is a bit more debatable and subjective and might fall into the category of more harm than good. But the intro sentences for sure are an easy fix, so done.